janusgraph-dist/src/assembly/static/bin/janusgraph-server.sh

```diff
@@ -254,8 +260,18 @@ startForeground() {
   fi
 }
 
+printConfig() {
+    echo "JANUSGRAPH_HOME: ${JANUSGRAPH_HOME}"
+    echo "JANUSGRAPH_YAML: ${JANUSGRAPH_YAML}"
+    echo "JANUSGRAPH_SERVER_CMD: ${JANUSGRAPH_SERVER_CMD}"
+    echo "LOG_DIR: ${LOG_DIR}"
+    echo "PID_FILE: ${PID_FILE}"
+    echo "JAVA_OPTIONS: ${JAVA_OPTIONS}"
```

Comment:
This should print the full `JAVA_OPTIONS` including everything from `jvm-11.options` / ``jvm-8.options`, right? I just build the distribution archive on my machine from this branch and then executed the script to get the output produced here, but the `JAVA_OPTIONS` don't look very complete:

```bash
$ bin/janusgraph-server.sh config
JANUSGRAPH_HOME: /mnt/c/Workspace/GitHub/janusgraph/janusgraph-dist/target/janusgraph-full-0.6.1-SNAPSHOT
JANUSGRAPH_YAML: /mnt/c/Workspace/GitHub/janusgraph/janusgraph-dist/target/janusgraph-full-0.6.1-SNAPSHOT/conf/gremlin-server/gremlin-server.yaml
JANUSGRAPH_SERVER_CMD: org.janusgraph.graphdb.server.JanusGraphServer
LOG_DIR: /mnt/c/Workspace/GitHub/janusgraph/janusgraph-dist/target/janusgraph-full-0.6.1-SNAPSHOT/logs
PID_FILE: /mnt/c/Workspace/GitHub/janusgraph/janusgraph-dist/target/janusgraph-full-0.6.1-SNAPSHOT/run/janusgraph.pid
JAVA_OPTIONS:  -XX:+CMSClassUnloadingEnabled  -javaagent:/mnt/c/Workspace/GitHub/janusgraph/janusgraph-dist/target/janusgraph-full-0.6.1-SNAPSHOT/lib/jamm-0.3.0.jar
COLLECTED_JAVA_OPTIONS_FILE:  -XX:+CMSClassUnloadingEnabled
```

This is still just the last line from the `OPTIONS` file `XX:+CMSClassUnloadingEnabled` together with the `javaagent`, isn't it?

Reply:
I will check again. Could be my last commit to fix codacy issues.